HHS/OPRE-OPS
# handles change of Service Requirement Type in wizard and BLI form refactor

## What changed

- handles change of Service Requirement Type in Create Agreement Wizard
- refactor BLI form CRUD operations
  - remove context provider 
- BLI state when in wizard
- handle cancelling out of wizard (if agreement was created in step2)
- UX improvements

## Issue

#1852 #2042 
#2070 

## How to test

1. Create a new Agreement via wizard. Complete Step 1.
2. Complete Step 2 (Agreement is created)
3. Add some SCs and BLIs.
4. Goto Step 2. // should allow you 
5. Go back to Step 3. // SCs and BLIs should still be there.

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
